interface/main/calendar/add_edit_event.php

```diff
@@ -191,6 +191,12 @@ function InsertEventFull()
         $pc_eid = InsertEvent($args);
         return $pc_eid;
     }
+
+    if (stristr($_POST['form_title'], 'telehealth')) {
```

Comment:
@juggernautsei Why restrict the appointment set event to telehealth? It would be a more useful event if you fire it off on every appointment add and let whoever is listening filter off of form_title etc.  
Also, I think the event would be more useful if you passed as much of the appointment data (such as the form_title) as possible to the event so that module writers can use that data.

Now if pid & form_date is all that is needed to hit the database and grab the appointment data then maybe that works, but it just seems a bit redundant to force the module creator to hit the DB to grab event data that you already have right here.

Reply:
I am writing this from a limited perspective of a request. I can open it up and pass in the $_POST data and it can be sorted by the module writer on their side. I will make that change and post it. Thanks for the suggestion.